draft-nottingham-public-resolver-errors.md

```diff
@@ -91,47 +125,62 @@ Note that there is no requirement for the template to expand to a URL on any par
 
 This section defines the data types used to look up the details of a filtering incident from a DNS error response. Note that these identifiers are not for presentation to end users.
 
-## DNS Resolver Operator ID {#op-id}
+## DNS Filtering Database Entry {#entry-id}
 
-A DNS Resolver Operator ID is a short, textual string that uniquely identifies the operator of a DNS resolver. It is carried in the EXTRA-TEXT field of the Extended DNS Error with the JSON field name "ro". For example:
+A Filtering Database ID is a short, textual string that uniquely identifies the
```

Comment:
```suggestion
A Filtering Database ID is a short string that uniquely identifies the
```
Or is a `string` different from a `textual string` somehow?

Reply:
@mnot I stole this from your original text---is there a difference?